assets/javascripts/kitten/components/form/text-input-with-limit.js

```diff
@@ -50,6 +59,6 @@ export class TextInputWithLimit extends React.Component {
 TextInputWithLimit.defaultProps = {
   tag: 'input',
   limit: 80,
-  defaultValue: "",
+  defaultValue: '',
```

Comment:
Je pense que oui vu que dans ce composant tu l'utilises pour ajouter la classe `is-disabled`. Même dans le cas où on s'en servait pas dans ce composant, je trouve ca intéressant de le mettre pour servir de documentation du composant (à défaut d'avoir une vraie doc).